### PR TITLE
Manipulate with cache metrics only if no force was supplied

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/user_api/openapi_server.py [hunk 1]
```diff
@@ -249,9 +249,8 @@ def expose_cache_hit_metrics(response):
                         metrics_cache_hit_adviser_unauthenticated.set(metrics_values.metric_cache_hit_adviser_unauth)
                 except Exception as metric_exc:
                     _LOGGER.error("Failed to set metric for adviser cache hits: %r", metric_exc)
-
-        if "provenance-checker" in data["analysis_id"]:
-            if data["cached"]:
+        elif data["analysis_id"].startswith("provenance-checker-"):
+            if data["cached"] and not data["parameters"]["force"]:
```
Comment (fridex) [inline thoth/user_api/openapi_server.py:253]: Based on code, force will always be false here.

Comment (fridex) [reply thoth/user_api/openapi_server.py:253]: https://github.com/thoth-station/user-api/blob/d5ae61985120f63bb80bf05063e8200c4af23ec1/thoth/user_api/api_v1.py#L126-L134

**Review by pacospace — APPROVED** [review]
lgtm! 💯

**Pull request merged**